doc/source/notebooks/advanced/kernels.pct.py

```diff
@@ -90,10 +90,10 @@ def plotkernelsample(k, ax, xmin=-3, xmax=3):
 # %% [markdown]
 # ## First example: create a Matern 3/2 covariance kernel
 #
-# Many kernels have hyperparameters, for example `variance` and `lengthscale`. You can change the value of these parameters from their default value of `1.0`.
+# Many kernels have hyperparameters, for example `variance` and `lengthscales`. You can change the value of these parameters from their default value of `1.0`.
```

Comment:
Would it be helpful to add a line here explaining how lengthscales as a float gives you *isotropic* lengthscale (for higher-dimensional input) and to pass in an *array of floats* to get ARD?

Reply:
Actually this is covered further down in the notebook